src/components/CheckboxGroup/CheckboxGroup.tsx

```
@@ -0,0 +1,80 @@
+
+      const handleLabelClick = useCallback(() => {
+        if (!tristate || disabled) return;
+        setTrivalue(curr => curr === null ? true : curr === true ? false : null);
+      }, [tristate, disabled]);
+
+      const LabelWrapper = useCallback(({ children }: PropsWithChildren) => {
+        if (!label) return children;
+        return <label className={labelCls} onClick={handleLabelClick}>
+          {children}
+          {label}
+        </label>;
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)